types/json-diff/index.d.ts

```diff
@@ -3,5 +3,9 @@
 // Definitions by: Tommy Wong <https://github.com/wchtommy20013/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function diff(obj1: any, obj2: any): any;
-export function diffString(obj1: any, obj2: any): any;
+export interface JsonDiffConfig {
+    color?: boolean;
+}
+
+export function diff(obj1: unknown, obj2: unknown): any;
```

Comment:
```suggestion
export function diff(obj1: unknown, obj2: unknown, config?: JsonDiffConfig): any;
```
IMO same applies to `diff`:
https://github.com/andreyvit/json-diff/blob/master/lib/index.coffee#L167

Reply:
I don't use the function so I didn't focus on it